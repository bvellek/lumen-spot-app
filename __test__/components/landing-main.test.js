import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LandingMain from '../../public/js/components/landing-main';

describe('<LandingMain />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LandingMain />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
