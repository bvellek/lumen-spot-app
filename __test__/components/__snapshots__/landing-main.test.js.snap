// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingMain /> it renders correctly 1`] = `
<main>
  <InfoWhat />
  <LandingImageSlider />
  <InfoLight />
  <LinkSection />
  <InfoAbout />
</main>
`;
